apps/frontend/src/pages/landing/libs/components/hero-section/styles.module.css

```
@@ -1,32 +1,46 @@
-.section {
+.hero-section {
 	display: flex;
 	flex-direction: column;
-	gap: 96px;
 	width: 100%;
-	padding-top: 96px;
 }
 
-.section-title {
+.hero-section-container {
+	display: flex;
+	flex-direction: column;
+	row-gap: 16px;
+	max-width: 1264px;
+	padding: 96px 32px;
+	margin: 0 auto;
+	text-align: center;
+}
+
+.title {
 	margin: 0;
 	font-size: var(--font-size-display);
 	font-weight: var(--font-weight-bold);
+	line-height: 1.1429;
 	color: var(--color-text-strong);
-	text-align: center;
 }
 
-.section-description {
+.description {
 	margin: 0;
 	font-size: var(--font-size-h4);
 	font-weight: var(--font-weight-regular);
+	line-height: 1.4;
 	color: var(--color-text-weak);
-	text-align: center;
 }
 
-.section-cover {
+.cover {
 	width: 100%;
 	aspect-ratio: 320 / 119;
 	background-image: url("/assets/images/landing-cover.jpg");
 	background-repeat: no-repeat;
 	background-position: center;
 	background-size: cover;
 }
+
+@media (width <= 768px) {
+	.hero-section-container {
+		padding: 50px 16px 30px;
```

let's slightly adjust it to make them divisible by 4
```suggestion
		padding: 48px 16px 32px;
```